transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/AccessLogConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package io.servicecomb.transport.rest.vertx.accesslog;
```

Comment:
Please add the license header.

Reply:
License header has been added.

Reply:
Please use  the new add Apache license, you can get it from the master code.